src/xcode/ENA/ENAUITests/ExposureSubmission/ExposureSubmissionUITests.swift

```diff
@@ -292,12 +292,16 @@ class ENAUITests_04a_ExposureSubmission: XCTestCase {
 		)
 		app.buttons["AppStrings.ExposureSubmissionDispatch.tanButtonDescription"].tap()
 
+		let continueButton = app.buttons["AppStrings.ExposureSubmission.primaryButton"]
+		XCTAssertTrue(continueButton.waitForExistence(timeout: .medium))
+		XCTAssertFalse(continueButton.isEnabled)
+
 		// Fill in dummy TAN.
-		XCTAssertTrue(app.buttons["AppStrings.ExposureSubmission.primaryButton"].waitForExistence(timeout: .medium))
 		type(app, text: "qwdzxcsrhe")
 
 		// Click continue button.
-		XCTAssertTrue(app.buttons["AppStrings.ExposureSubmission.primaryButton"].isEnabled)
+		XCTAssertTrue(continueButton.waitForExistence(timeout: .medium))
+		XCTAssertTrue(continueButton.isEnabled)
```

Comment:
Even cooler:
```suggestion
		XCTAssertTrue(continueButton.isHittable)
```

Reply:
(and following)

Reply:
i will habe a try if it does work without flakiness, too.